### PR TITLE
New build

### PR DESCRIPTION
**Pull request by aizerin**
Closes #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,27 +20,46 @@ $ yarn add ramda ramda-extension
```
Comment (tommmyy) [inline README.md]: usage with require() are not necessary

File: jest.config.json [hunk 2]
```diff
@@ -1,10 +1,3 @@
 {
-  "rootDir": "./src",
-  "cacheDirectory": "/tmp/jest_cache",
-  "testPathDirs": [
-    "<rootDir>"
-  ],
-  "modulePathIgnorePatterns": [
-    "/build/"
-  ]
+  "rootDir": "./source"
```
Comment (tommmyy) [inline jest.config.json]: I know that ramda use "source" but "src" is more standard name

File: rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+const banner = `
+//  Ramda-extension v${require('./package.json').version}
+//  https://github.com/tommmyy/ramda-extension
+//  (c) 2017-${new Date().getFullYear()}
+//  Ramda-extension may be freely distributed under the MIT license.
+`;
+
+const config = {
+	input: 'source/index.js',
```
Comment (tommmyy) [inline rollup.config.js]: I know that ramda use "source" but "src" is more standard name

**Comment by aizerin** [comment]
I also like src more :). I will rename it back and use lib instead of src for cjs build, which is also widely used.

**Pull request merged**